### DISCUSSION TITLE
FLC and Speed not working?

### DISCUSSION THREAD

**Discussion by BravoAlfaZulu** [calibx/msfsdeck#210] [Q&A]
I can't seem to get Autopilot FLC or Speed Modes to work in any aircraft (C172, TBM, Kodiak 100, A320 etc) is this bugged or am I doing something wrong? (Pressing the buttons on the G1000 in the sim with the mouse work).
Is the button using a key assignment or is it interacting with the sim directly? (There is currently no key binding assigned for FLC mode)

**Comment by calibx** [comment]
Hi,
Sometimes the default implementation doesn't works with all aircrafts. I will try to investigate.
The communication between the plugin an MSFS doesn't use keybindings so you can set them as you want.
